README.md

```diff
@@ -27,27 +27,41 @@ Add the following to your `project/plugins.sbt` or `~/.sbt/0.13/plugins/plugins.
 
 additionally, use one of the older README.md files: (https://github.com/sbt/sbt-git/blob/v0.7.1/README.md)
 
-### Using JGit ###
+### JGit ###
 
-If you do not have git installed and available on your path (e.g. you use windows),
-make sure your `git.sbt` or `~/.sbt/0.13/git.sbt` file looks like this:
+JGit is a Java interface to git that allows some git operations to be
+performed in the JVM without invoking an external git executable. By default,
+this plugin uses JGit for read-only operations such as inspecting HEAD; for
+write operations, it assumes a git executable is present and on the PATH and
+it uses that.
+
+In certain circumstances you may want to force the use of JGIT or an
+executable for both read-only and read-write operations; for example, if you
+have no git executable installed (e.g. you use windows) you need to disable
```

Comment:
can't you have git installed on windows? https://git-scm.com/downloads suggests so

Reply:
the example is lifted from the passage i replaced which justifies using `useJGit` option. of course windows users should be able to install git and add it to the PATH, the situation where a Scala dev (much less a user of this repo) hasn't done so does seem kind of far-fetched in this day and age. nonetheless *some* example seems appropriate and you could do worse than "windows user that can't be bothered to add `git` to $PATH".

that said it's not my doc so, whatever you prefer!

Reply:
ah, I missed that. that shouldn't hold back this PR then, even though it still looks somewhat odd to me ;) :+1: